tools/bpf2c/bpf_code_generator.cpp

```diff
@@ -153,7 +155,7 @@ bpf_code_generator::program_sections()
     std::vector<std::string> section_names;
     for (const auto& section : reader.sections) {
         std::string name = section->get_name();
-        if (name.empty() || name[0] == '.')
+        if (name.empty() || (section->get_size() == 0))
```

Comment:
I think we should have the same logic here as is in `ebpf_api.cpp`. We should consider ".text" section for emitting and verification only when there are no other programs?

Reply:
Done